colossalai/zero/sharded_model/sharded_model_v2.py

```diff
@@ -76,7 +76,9 @@ def __init__(self,
                  fp32_reduce_scatter: bool = False,
                  tensor_placement_policy: str = 'cuda',
                  gradient_predivide_factor: Optional[float] = 1.0,
-                 reuse_fp16_shard: bool = False):
+                 reuse_fp16_shard: bool = False,
+                 *args,
```

Comment:
It is never used. Remove it?

Reply:
I think it doesn't matter. leaving it here is harmless. Also *args and **kwargs often come together to make the parameters passed in scalable.

Reply:
`*args` is usaully in front of kwargs like `xxx=xxx`